pandas/tests/window/test_rolling.py

```diff
@@ -1950,3 +1950,24 @@ def test_numeric_only_corr_cov_series(kernel, use_arg, numeric_only, dtype):
         op2 = getattr(rolling2, kernel)
         expected = op2(*arg2, numeric_only=numeric_only)
         tm.assert_series_equal(result, expected)
+
+
+def test_rolling_rolling_sum_window_microseconds_confilict_timestamp():
```

Comment:
When generalizing the fix for all resolutions, we can update this test to be parametrized across all resolutions:

```

```suggestion
@pytest.mark.parametrize("unit", ["s", "ms", "us", "ns"])
def test_rolling_rolling_sum_window_microseconds_confilict_timestamp(unit):
```

and then this `unit` variable can be used in `as_unit` or when constructing the object

Reply:
Great! I’ll fix it soon :)